docs/data/charts/scatter/scatter.md

```diff
@@ -116,3 +116,13 @@ See [Axis—Grid](/x/react-charts/axis/#grid) documentation for more information
 ### Shape 🚧
 
 ### Size 🚧
+
+## Plot Customization
+
+You can customize the plotting of the data in a scatter chart by providing custom components as `children` of the `ScatterChart` component.
+
+A scatter chart's series can be accessed through the `useScatterSeries` hook. This hook returns the order of the series and information about the series themselves, including their data points, color, etc.
+
+See [Custom components](/x/react-charts/components/) to learn how to further customize your charts.
+
+{{"demo": "CustomScatter.js"}}
```

Comment:
@alexfauquette not sure if this is enough information. Do you think I should expand on any particular topic?

Reply:
Thats a good starting point. We can improve later based on what users open issues for :+1: